bundletester/tester.py

```diff
@@ -83,6 +83,9 @@ def configure():
                         help='A plan to deploy charm under')
     parser.add_argument('--deploy-budget',
                         help='Deploy budget and allocation limit')
+    parser.add_argument('--no-chaos', action="store_true",
```

Comment:
I mentioned in this #103, but I'll repeat it here.. `--no-chaos` implies to me that only the chaotic matrix tests will be skipped.  In fact, this flag is turning off all matrix tests.  I would prefer this flag be called `--no-matrix`.